<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PDF Tools TaskDental</string>
  </property>
  <property name="accessibleName">
   <string>PdfTools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>509</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pbPrevPage">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&lt; [F1]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPageNum">
       <property name="text">
        <string>--/--</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbNextPage">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>[F2]&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lblImagePdf">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>341</width>
      <height>451</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Arrastra un pdf aqui para procesarlo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="SeccionesFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>381</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="group02">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>361</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Sección</string>
     </property>
     <widget class="QLabel" name="lblPagFin">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pag. Fin</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>90</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pbAdd">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Añadir</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbTipoDoc">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Historia Clínica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Alta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Protección de datos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Presupuesto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Afiliación Tarjeta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.Endodoncia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.AnestesiaLocal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.ExtraccCordales</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.Odont.Conserva</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.ProtesisFijaImplant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.TrataImplantes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.ObturaDental</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.TrataDental</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.ColocaProtesisDental</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.InjertoHueso</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.ElevaSeno</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.ExtracSimple</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cons.Reendo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ref.Lab.Externo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RadioAnalógica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JustificanteCobro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DevoluciónPaciente</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CuestionarioCovid19</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DiagnosticoOrtodoncia</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Docu.Grafica</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Financiación</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Historial Quirófano</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="lbFecha">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha asociada</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbPagFin">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
     <widget class="QLabel" name="lblPagIni">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pag. Inicio</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblTipoDoc">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>23</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo de documento</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sbPagIni">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>998</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbSinFecha">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Sin fecha</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>361</width>
       <height>171</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>361</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>72</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>PagInicio</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PagFin</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>TipoDoc</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fecha</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>403</x>
      <y>10</y>
      <width>20</width>
      <height>541</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnProcess">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>493</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Procesar PDF</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInfo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Información</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="group01">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Datos Principales</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>18</x>
       <y>20</y>
       <width>351</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formDatosPacientes">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <property name="rowWrapPolicy">
       <enum>QFormLayout::DontWrapRows</enum>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Nombre Fichero</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblOriginalFileName">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Numero Paciente          </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="nombrePaciente">
        <property name="text">
         <string>Nombre Paciente          </string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblNombrePacienteDB">
        <property name="text">
         <string> ..............................................................</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSpinBox" name="sbPaciente">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <bold>true</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::PlusMinus</enum>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>15000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnConfNum">
          <property name="text">
           <string>CONFIRMAR</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAction">
    <property name="title">
     <string>Acciones</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionConfig"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuAction"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPreferencias">
   <property name="text">
    <string>Preferencias</string>
   </property>
  </action>
  <action name="actionAbrir_PDF">
   <property name="text">
    <string>Abrir PDF</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reiniciar</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Configuración</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>sbPaciente</tabstop>
  <tabstop>btnConfNum</tabstop>
  <tabstop>cbTipoDoc</tabstop>
  <tabstop>sbPagIni</tabstop>
  <tabstop>sbPagFin</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>rbSinFecha</tabstop>
  <tabstop>pbAdd</tabstop>
  <tabstop>btnProcess</tabstop>
  <tabstop>pbPrevPage</tabstop>
  <tabstop>pbNextPage</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
